libs/grout/test-utils/src/mock_client.test.ts

```diff
@@ -41,7 +41,7 @@ test('catches exceptions from mock function failures and logs them', async () =>
 });
 
 test('doesnt catch intentional exceptions from mock functions', () => {
-  const mockClient = createMockClient<typeof api>();
+  const mockClient = createMockClient<typeof api>({ catchErrors: true });
```

Comment:
While the test description makes the expected behavior clear, `catchErrors: true` sounds like a setting for catching all errors, intentional or not. `catchUnexpectedErrors: true` (or something like that) might be clearer

Reply:
Will change!